source/default/_patterns/00-protons/index.js

```diff
@@ -1,9 +1,32 @@
 /**
- * Base css generation and global js logic
+ * Base css generation and global js logic.
  */
 
 import './_base.scss';
 
+import { mediaBreakpoint } from 'protons/utilities';
+import scssVariables from '../../_data/scssVariables.json';
```

Comment:
The thing that makes this `sassVariables.json` is a Sass plugin, generating JSON meant exclusively for PL to show off. It's straightforward way to get sass variables -> json -> html.

With enquire, can we make it not dependent on this json blob? I'm ok pushing the logic to runtime by reading the `--variables` on `<html>`. The reason I don't want to be dependent on this blob is that Pattern Lab might not be generated in CI when it's time for Drupal to roll around and compile its own assets out of the design system.

Reply:
Ah, sure. I'm not sure of the best way to ensure that the breakpoints are equal between SCSS and JS, then, other than to just manually configure them (which is, naturally, how I did it in the previous project where I grew to love Enquire).

I'll update to have those defined elsewhere.

Reply:
Just read those breakpoints off the `--variables` on `<html>`. Those are all set via Sass -> CSS, so the JS would be reading the "single source of truth" here.

Reply:
@illepic, yeah, I just hadn't had the time until I started an hour ago. PR updated.